### ISSUE TITLE
Graphicscache not updated when input files are edited

### ISSUE THREAD

**Issue by MatthiasNieuwenhuisen** [xqms/graphicscache#3]
If an already cached image file is edited, the cache is not updated when recompiling the LaTeX document.

**Comment by xqms** [comment]
Hmm, I can't reproduce that here. The update check is based on the file modification time. Does your filename contain strange characters?

**Comment by MatthiasNieuwenhuisen** [comment]
It seems that this happens if the file extension is not speficied.

In this example the first figure is updated as expected, while the second figure is not.

```latex
\documentclass{article}
\usepackage{graphicx}
\usepackage{graphicscache}
\begin{document}
\includegraphics[width=\linewidth]{left-fig.jpg}
\includegraphics[width=\linewidth]{left-fig}
\end{document}
```

**Comment by MatthiasNieuwenhuisen** [comment]
Related:
[Graphics file extensions and their order of inclusion when not specified](https://tex.stackexchange.com/questions/72930/graphics-file-extensions-and-their-order-of-inclusion-when-not-specified)

**Comment by xqms** [comment]
Here is code for finding the filename in the same manner as \includegraphics:

https://tex.stackexchange.com/questions/99070/check-for-a-valid-file-before-using-includegraphics/99099#99099

**Comment by xqms** [comment]
Should be fixed in c4fa1b9.
